### DISCUSSION TITLE
Setting API key in Azure

### DISCUSSION THREAD

**Discussion by thebrahman** [qdrant/qdrant#2730] [General]
Hi, I have successfully run Qdrant in docker locally and can easily edit the config to include an API key. 

I have used this template to install in azure:
https://github.com/Azure-Samples/qdrant-azure/tree/main/Azure-Container-Instances

there is persistent storage, but not of the config files. does anyone know how i can access the config files to add an API key?

**Comment by thebrahman** [comment]
am running in VM in meantime

**Comment by azayarni** [comment]
Please ask on our Discord Community Server https://qdrant.to/discord
